### PR TITLE
Quiz Result page is ready for reviews

### LINKED ISSUE #19: Storybook: Career Quiz Result Page Component
## User story

**Who:** **As a** user
**What:** **I want to** be open the Career Quiz Result Page
**Why:** **so that I can** see the results of the QUIZ.

## Acceptance criteria

Create a storybook component for the Career Quiz Result Page where we aggregate all the components below:

- Robot logo and a label below with the text BASED ON QUIZ RESULTS
- HEADER - https://github.com/HackYourFuture-CPH/Guide-IT/issues/13
- CARD for YOUR PERSONAL PROFILE - https://github.com/HackYourFuture-CPH/Guide-IT/issues/16
- CARD for YOUR PROFESSIONAL PROFILE - https://github.com/HackYourFuture-CPH/Guide-IT/issues/16
- TEASER - https://github.com/HackYourFuture-CPH/Guide-IT/issues/17
- BUTTON - https://github.com/HackYourFuture-CPH/Guide-IT/issues/18

 
These are the 3 different options for the career quiz result page can look like

Screenshot 2021-04-13 at 15 37 38
Screenshot 2021-04-13 at 15 37 28
Screenshot 2021-04-13 at 15 37 17

Check the wireframes here: https://www.figma.com/file/9SJRaxPSuPI3lSl8ULNyJ8/GuideIT-For-Developers?node-id=0%3A1

## Implementation details

Use the image Vector.svg located in this folder **/src/client/assets/images/** as a background image and find a way for half of the it to have a layer of opacity like we can see in the screenshot above.

### PR DESCRIPTION
**Pull request by aijazraja**
# Description

The Quiz Result page is ready for reviews

Fixes #19 

# How to test?

Please use Storybook to test

# Checklist

- [x ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/containers/QuizResultPage/QuizResultPage.component.js [hunk 1]
```diff
@@ -1,3 +1,73 @@
...
+                  BASED ON <br />
+                  QUIZ <br />
+                  RESULTS
+                </p>
+                {/* <RobotLogo /> */}
+              </div>
+              <div className="left-div-card">
+                <CardProfileResultComponent
+                  title="Your Personal Profile"
+                  results={resultpersnol}
```
Comment (cph-kiwi) [inline src/client/containers/QuizResultPage/QuizResultPage.component.js:50]:
```suggestion
                  results={resultpersonal}
```

File: src/client/containers/QuizResultPage/QuizResultPage.component.js [hunk 2]
```diff
@@ -1,3 +1,73 @@
...
+import RegisterTeaser from '../../components/RegisterTeaser/RegisterTeaser';
+import PageHeader from '../../components/PageHeaderComponent/PageHeader';
+import { gray, greenBright } from 'chalk';
+export const QuizResultPage = () => {
+  const result = [
+    'creative',
+    'knows fundamental database concepts',
+    'works well under pressure',
+  ];
+  const resultpersnol = [
```
Comment (cph-kiwi) [inline src/client/containers/QuizResultPage/QuizResultPage.component.js:16]:
```suggestion
  const resultpersonal = [
```

File: src/client/containers/QuizResultPage/QuizResultPage.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+.right-div {
+  padding-left: 35px;
+}
+
+.rigt-div-logo {
+  padding-left: 255px;
+  padding-bottom: 15px;
+}
```
Comment (cph-kiwi) [inline src/client/containers/QuizResultPage/QuizResultPage.css:45]: I would like to see this in storybook before commenting on the CSS. However I can't open it in storybook as I'm getting errors around redi-logo.png, in SidebarMenu.js and ResultJobFeatures.js.

**Review by cph-kiwi — CHANGES_REQUESTED** [review]

**Review by qamarfj — COMMENTED** [review]
I think you added some extra border.
page

File: src/client/containers/QuizResultPage/QuizResultPage.css [hunk 4]
```diff
@@ -0,0 +1,45 @@
+.main-div {
```
Comment (cph-kiwi) [inline src/client/containers/QuizResultPage/QuizResultPage.css:1]: I think the name of this file needs changing to QuizResultPage.styles.css, as required in this guidelines page. https://github.com/HackYourFuture-CPH/Guide-IT#coding-guidelines-for-components

File: src/client/containers/QuizResultPage/QuizResultPage.component.js [hunk 5]
```diff
@@ -1,3 +1,75 @@
...
+              <RegisterTeaser signInURL="#" />
+            </div>
+          </div>
+          <div className="right-div">
+            <div className="right-div-card">
+              <CardProfileResultComponent
+                title="Your Professional Profile"
+                results={result}
+              />
+            </div>
```
Comment (cph-kiwi) [inline src/client/containers/QuizResultPage/QuizResultPage.component.js:66]: I think a teaser (https://github.com/HackYourFuture-CPH/Guide-IT/issues/17) was supposed to go in here?

**Review by cph-kiwi — CHANGES_REQUESTED** [review]

**Review by bytewiz — APPROVED** [review]
Looks good but be careful with large numbers of fixed values in the CSS such as `padding-left: 258px` as they might not be very responsive for the different screen sizes. Just keep it in mind :)

**Comment by aijazraja** [comment]
> Looks good but be careful with large numbers of fixed values in the CSS such as `padding-left: 258px` as they might not be very responsive for the different screen sizes. Just keep it in mind :)

yea  I will take care of it... can you please approve my PR now :)

File: src/client/containers/QuizResultPage/QuizResultPage.component.js [hunk 6]
```diff
@@ -1,3 +1,75 @@
 import React from 'react';
+import './QuizResultPage.css';
+import background from '../../assets/images/Vector.svg';
+import Robot from '../../assets/images/robot_logo.png';
+import RobotLogo from '../../components/RobotLogo/RobotLogo';
+import CardProfileResultComponent from '../../components/CardProfileResult/CardProfileResultComponent.js';
+import RegisterTeaser from '../../components/RegisterTeaser/RegisterTeaser';
```
Comment (qamarfj) [inline src/client/containers/QuizResultPage/QuizResultPage.component.js:7]:
```suggestion
import RegisterTeaser from '../../components/RegisterTeaser/RegisterTeaser.component';
```

File: src/client/containers/QuizResultPage/QuizResultPage.component.js [hunk 7]
```diff
@@ -1,3 +1,75 @@
 import React from 'react';
+import './QuizResultPage.css';
+import background from '../../assets/images/Vector.svg';
+import Robot from '../../assets/images/robot_logo.png';
+import RobotLogo from '../../components/RobotLogo/RobotLogo';
+import CardProfileResultComponent from '../../components/CardProfileResult/CardProfileResultComponent.js';
```
Comment (qamarfj) [inline src/client/containers/QuizResultPage/QuizResultPage.component.js:6]:
```suggestion
import CardProfileResultComponent from '../../components/CardProfileResult/CardProfileResult.component.js';
```

File: src/client/containers/QuizResultPage/QuizResultPage.component.js [hunk 8]
```diff
@@ -1,3 +1,75 @@
 import React from 'react';
+import './QuizResultPage.css';
+import background from '../../assets/images/Vector.svg';
+import Robot from '../../assets/images/robot_logo.png';
+import RobotLogo from '../../components/RobotLogo/RobotLogo';
```
Comment (qamarfj) [inline src/client/containers/QuizResultPage/QuizResultPage.component.js:5]:
```suggestion
import RobotLogo from '../../components/RobotLogo/RobotLogo.component';
```

File: src/client/containers/QuizResultPage/QuizResultPage.component.js [hunk 9]
```diff
@@ -1,3 +1,75 @@
...
+import CardProfileResultComponent from '../../components/CardProfileResult/CardProfileResultComponent.js';
+import RegisterTeaser from '../../components/RegisterTeaser/RegisterTeaser';
+import PageHeader from '../../components/PageHeaderComponent/PageHeader';
```
Comment (qamarfj) [inline src/client/containers/QuizResultPage/QuizResultPage.component.js:8]:
```suggestion
import PageHeader from '../../components/PageHeaderComponent/PageHeader.component';
```

**Review by qamarfj — APPROVED** [review]

**Pull request merged**